Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -1091,7 +1175,14 @@ std::vector<std::vector<unsigned int>> getMolSelfMatches(
 }  // end of namespace detail
 
 void EmbedMultipleConfs(ROMol &mol, INT_VECT &res, unsigned int numConfs,
-                        const EmbedParameters &params) {
+                        EmbedParameters &params) {
+  if (params.trackFailures) {
+#ifdef RDK_BUILD_THREADSAFE_SSS
+    std::lock_guard<std::mutex> lock(GetFailMutex());
+#endif
+    params.failures.resize(10);
```

Comment:
Is there a way to not hardcode the size of failures?

Reply:
We could either use a map or check the size of the failures vector each time it's used and resize if necessary.
Either seems kind of overkill.

To make it a bit more obvious that this exists (and what its value is), I'll make a constant `MAX_TRACKED_FAILURES`

Reply:
That makes sense